src/accordion/panel.js

```diff
@@ -90,62 +113,32 @@ class Panel extends React.Component<PanelPropsT, {isFocusVisible: boolean}> {
     } = this.props;
 
     const sharedProps = this.getSharedProps();
-    const {
-      PanelContainer: PanelContainerOverride,
-      Header: HeaderOverride,
-      Content: ContentOverride,
-      ToggleIcon: ToggleIconOverride,
-    } = overrides;
-
-    const isIconOverriden =
-      ToggleIconOverride &&
-      (typeof ToggleIconOverride === 'function' ||
-        typeof ToggleIconOverride.component === 'function');
-
-    const [PanelContainer, panelContainerProps] = getOverrides(
-      PanelContainerOverride,
-      StyledPanelContainer,
-    );
-    const [Header, headerProps] = getOverrides(HeaderOverride, StyledHeader);
-    const [Content, contentProps] = getOverrides(
-      ContentOverride,
-      StyledContent,
-    );
-    const [ToggleIcon, toggleIconProps] = getOverrides(
-      ToggleIconOverride,
-      StyledToggleIcon,
-    );
-
-    const toggleIconOverrides = mergeOverrides(
-      {Svg: ToggleIcon},
-      // $FlowFixMe
-      {Svg: ToggleIconOverride},
-    );
+    const ToggleIconComponent = expanded ? CheckIndeterminateIcon : PlusIcon;
 
-    // it's a bit tricky ¯\_(ツ)_/¯
-    // we only want to use the theme overrides, if it was not override locally
-    const ToggleIconComponent = isIconOverriden
-      ? ToggleIcon
-      : expanded
-      ? CheckIndeterminateIcon
-      : PlusIcon;
+    const HeaderOverrides = mergeOverrideResources(overrides.Header || {}, {
+      props: {
+        // constrain focus handlers, overrides handled in component methods
+        onFocus: this.handleFocus,
+        onBlur: this.handleBlur,
```

Comment:
I think `onClick={this.onClick}` and `onKeyDown={this.onKeyDown}` should be included in here also.